src/style.css

```
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 800;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+  --height-search-input--big: 70px;
+  --height-search-input--small: 42px;
+}
+
+.search-bar {
+  position: relative;
+}
+
+.search-field {
+  margin-bottom: 20px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  width: 100%;
+}
+
+.search-field--big {
+  height: var(--height-search-input--big);
+  padding-left: 62px;
+  font-size: 16px;
+
+}
+
+.search-field--small {
+  height: var(--height-search-input--small);
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search-svg {
+  position: absolute;
+}
+
+.search-svg--big {
+  padding-left: 26px;
+  top: 25px;
+  bottom: 26px;
+  width: 19px;
```

```suggestion
  left: 26px;
  top: 25px;
  bottom: 26px;
  width: 19px;
  height: 19px
```
it can help with image
use `left` instead padding to position the element, and specify height